src/features/PathAutocompleteProvider.ts

```diff
@@ -147,6 +146,52 @@ export class PathAutocomplete implements vs.CompletionItemProvider {
         return path.join(currentDir, insertedPath);
     }
 
+    getInsertedPath(currentLine: string, currentPosition: number): string {
+        var lastQuote = -1;
+        var insideSingleQuote = false;
+        var insideDoubleQuote = false;
+        var insideBacktickQuote = false;
+
+        var lastSpace = -1;
+
+        for (var i = 0; i < currentPosition; i++) {
+            var c = currentLine[i];
+
+            // skip next character if escaped
+            if (c == "\\") {
+                i++;
+                continue;
+            }
+
+            // handle space
+            if (c == " ") {
+                lastSpace = i;
+                continue;
+            }
+
+            // handle quotes
+            if (c == "'") {
```

Comment:
Feel free to change it, I tried to keep the behavior for quotes as close to the original as possible.